apps/bbsync/query.py

```diff
@@ -5,6 +5,46 @@
 from osidb.models import Flaw, FlawImpact, PsModule
 
 
+class SRTNotesBuilder:
+    """
+    Bugzilla flaw bug SRT notes field JSON content builder
+    """
+
+    def __init__(self, flaw, old_flaw=None):
+        """
+        init stuff
+        parametr old_flaw is optional as there is no old flaw on creation
+        and if not set we consider the query to be a create query
+        """
+        self.flaw = flaw
+        self.old_flaw = old_flaw
```

Comment:
At this stage it is really unused and not documented as I mostly just copied the structure of the existing query builder class. It will be used in the next followup PR to get differences between the current and the previous state of the flaw. Sorry for not explaining but if you are not strongly against it I would just leave it there. I am already working on the followup which should use it :angel: